src/amqp_uri.erl

```diff
@@ -231,6 +232,11 @@ parse_amqp_param(Field, String) ->
 find_path_parameter(Value) ->
     find_identity_parameter(Value).
 
+find_sni_parameter("disable") ->
```

Comment:
According to http://erlang.org/doc/man/ssl.html this should be `disable` (an atom)?

Reply:
Right - it will come in as a string from the amqps URI, and then `find_sni_parameter` will convert it to an atom. Of course this means you can't name your server `disable`...